docs/developers_guide/organization/steps.md

```diff
@@ -357,7 +357,36 @@ attributes.  Another reason might be to set these attributes using an algorithm
 When overriding `constrain_resources`, it is important to also call the base
 class' version of the method with `super().constrain_resources()`.
 
-(dev-step-setup)=
+The names of the resources are related to the
+[Slurm](https://slurm.schedmd.com/srun.html) naming conventions:
+
+``ntasks``
```

Comment:
@altheaden, this is great!  If you just convert the double back quotes to single back quotes, that should be all that's left.  I'll have to check the glossary-style formatting with a local build once that's done.

Reply:
Yeah, I did a local build and the glossary-style formatting looks good!

The double back-quotes are fine, too, but I prefer to have them as single back-quotes just for consistency.

Reply:
That's fine, I can make them single quotes. I agree with consistency.

Reply:
Perfect, thanks!